kong/runloop/handler.lua

```diff
@@ -672,8 +672,8 @@ do
 end
 
 
-local function balancer_setup_stage1(ctx, scheme, host_type, host, port,
-                                     service, route)
+local function balancer_prepare(ctx, scheme, host_type, host, port,
```

Comment:
The `balancer_setup_stage2` is still there, perhaps that needs to be renamed to `balancer_execute` now too?